src/core/socket.c

```diff
@@ -1941,11 +1941,13 @@ static int socket_coldplug(Unit *u) {
 static int socket_spawn(Socket *s, ExecCommand *c, PidRef *ret_pid) {
 
         _cleanup_(exec_params_clear) ExecParameters exec_params = {
-                .flags     = EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_APPLY_TTY_STDIN,
-                .stdin_fd  = -EBADF,
-                .stdout_fd = -EBADF,
-                .stderr_fd = -EBADF,
-                .exec_fd   = -EBADF,
+                .flags            = EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_APPLY_TTY_STDIN,
+                .stdin_fd         = -EBADF,
+                .stdout_fd        = -EBADF,
+                .stderr_fd        = -EBADF,
+                .exec_fd          = -EBADF,
+                .bpf_outer_map_fd = -EBADF,
+                .user_lookup_fd   = -EBADF,
```

Comment:
hmm, if we keep adding fields here, this becomes fragile. maybe it's time to add a helper macro:

```c
#define EXEC_PARAMETERS_INIT(_flags) \
        (ExecParameters) { \
                .flags = (_flags), \
                .stdin_flags = -EBADF, \
                … 
        };
```

and then use this here?

shortens the code and if we keep adding fields to the struct we don't need to manually update each location anymore.